Formula/eksctl.rb

```diff
@@ -0,0 +1,36 @@
+class Eksctl < Formula
+  desc "The official CLI for Amazon EKS"
+  homepage "https://eksctl.io"
+  url "https://github.com/weaveworks/eksctl.git",
+      :tag      => "0.24.0",
+      :revision => "f23f8d7d75943d1a62c87ce2f3d455da7af741fa"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+  depends_on "go-bindata" => :build
+  depends_on "aws-iam-authenticator"
+  depends_on "kubernetes-cli"
+
+  def install
+    # Some dependencies expect go-bindata to be located in the GOPATH. We want to use the global one instead
+    inreplace "Makefile", "$(gopath)", HOMEBREW_PREFIX.to_s
```

Comment:
`GOBIN` seems to be overridable in Makefile, maybe try something like this:

```suggestion
    ENV["GOBIN"] = HOMEBREW_PREFIX/"bin"
```

Reply:
@dawidd6 It works too